FAQ.md

```diff
@@ -124,3 +124,21 @@ for i in ${userlist[@]};do
     MENU_OPTION=1 CLIENT=$i PASS=1 ./openvpn-install.sh
 done
 ```
+
+---
+
+**Q:** How do I change the default `.ovpn` file created for future clients?
```

Comment:
@randomshell @angristan  added general instructions for the `client-template.txt` file